schemas/app-sre/scorecard-2.yml

```diff
@@ -2,6 +2,15 @@
 "$schema": /metaschema-1.json
 version: '1.0'
 type: object
+title: "Scorecard"
+description: |
+  Schema for defining a self-service scorecard for limited-scope service engagements.
```

Comment:
```suggestion
  Schema for defining a service scorecard for limited-scope service engagements.
```

Reply:
This is funny to me! I reviewed all git history for every schema as I developed the documentation. The context for this schema is "a la carte" services vs fully fledged services. 

cc https://issues.redhat.com/browse/APPSRE-6506 "These scorecards are essentially self-assessments, to be **filled in by the service owners and to be kept up to date by them**.
Anyone at Red Hat should be able to see this maturity scorecard."

This schema seems purpose-built as alternative to SRE checkpoints.

Reply:
cc https://github.com/app-sre/qontract-schemas/pull/280 and https://github.com/app-sre/qontract-schemas/pull/287

Reply:
we will probably extend this a bit in the context of dedicated SRE. Let's review at that time